src/idiom162.fuzion

```diff
@@ -0,0 +1,16 @@
+ex162 is
+  bat =>
+    { }
```

Comment:
Remove `{ }`
Needs a bit of time to get used to seeing nothing for `unit`, but this is what I currently prefer.